packages/calypso-analytics/src/tracks.ts

```diff
@@ -108,6 +108,13 @@ function checkForBlockedTracks(): Promise< void > {
 	} );
 }
 
+/**
+ * Returns a promise that marks whether and when the external Tracks script loads.
+ */
+export function getTracksLoadPromise() {
+	return _loadTracksResult
+}
```

Comment:
@scinos might get you to check this as I am not familiar with this package.

Reply:
Do I need to bump the package version? I am not familiar with how the package deps update.

Reply:
No need to bump the package version if you just want to use this new functionality inside the monorepo.

We only need to bump the version if/when we are going to do a public release of the package. I can see the latest release was done by @sarayourfriend ~7 months ago, maybe she knows more about this package?

Reply:
Sorry, I don't know anything about the package, I just released it as part of a mass release of packages that needed releasing.